.gitlab-ci.yml

```diff
@@ -1,13 +1,25 @@
 stages:
+  - build
   - test_prep
   - test
-  - build
   - publish
   - trigger_prep
   - trigger
 
 variables:
-  GOLANG_VERSION: "1.17.13"
+  GOLANG_VERSION:
+    description: "The version of the image: `docker.io/library/golang` to use for testing"
+    value: "1.21.1"
```

Comment:
Careful about this. This exact variable is parsed by the deb package build system to use exact this version for building.

It is a long story ([here](https://northerntech.atlassian.net/browse/QA-614) and [here](https://github.com/mendersoftware/mender-connect/pull/104/files)), but please use a different name.